keter-backend/src/main/java/ru/avplatonov/keter/backend/controllers/management/create/CreateGraph.java

```diff
@@ -0,0 +1,50 @@
+package ru.avplatonov.keter.backend.controllers.management.create;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ru.avplatonov.keter.backend.initialize.GraphTemplate;
+import ru.avplatonov.keter.backend.initialize.NodeTemplate;
+
+import java.io.IOException;
+import java.util.*;
+
+import static ru.avplatonov.keter.backend.Application.graphsDB;
+import static ru.avplatonov.keter.backend.Application.nodesDB;
+
+@RestController
+@EnableAutoConfiguration
+public class CreateGraph {
+
+    @RequestMapping(value = "/create/graphs",
+            headers = {"Content-type=application/json"})
+    public String service(
+            @RequestBody GraphTemplate graphTemplate
+    ) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
As I know you can use only one ObjectMapper
why you create this object on every request?